app/routines/newflow/verify_educator.rb

```diff
@@ -0,0 +1,88 @@
+module Newflow
+  # Verify their Educator status given their (SheerID) Verification ID
+  # Expects a user to already exist with that SheerID Verification ID
+  class VerifyEducator
+    lev_routine active_job_enqueue_options: { queue: :educator_signup_queue }, use_jobba: true
+    uses_routine UpsertSheeridVerification
+    uses_routine UpdateSalesforceLead
+
+    protected ###############
+
+    def exec(verification_id:, user:)
+      status.set_job_name(self.class.name)
+      status.set_job_args(verification_id: verification_id.to_s, user: user.to_global_id.to_s)
+
```

Comment:
is all this unit tested?

Reply:
This is part of the Lev gem which we own and that gem is unit tested. This part doesn't anything besides making more informative log entries, I believe, also the reason why it wasn't unit tested in the legacy flow which has been running for years.